debian/marquee/nginx/sites/whatwg.org.conf

```diff
@@ -121,34 +121,36 @@ server {
     location ~ ^/specs/web-apps/2007-10-26/multipage/images(/(.*))?$ {
         return 301 /specs/web-apps/2007-10-26/images/$2;
     }
-    location /specs/web-apps/current-work/ {
-        location ~ ^/specs/web-apps/current-work/complete(/(.*))?$ {
-            return 301 https://html.spec.whatwg.org/multipage/$2;
-        }
-        location = /specs/web-apps/current-work/html-a4.pdf {
-            return 301 https://html.spec.whatwg.org/print.pdf;
-        }
-        location = /specs/web-apps/current-work/html-letter.pdf {
-            return 301 https://html.spec.whatwg.org/print.pdf;
-        }
-        location = /specs/web-apps/current-work/html5-a4.pdf {
-            return 301 https://html.spec.whatwg.org/print.pdf;
-        }
-        location = /specs/web-apps/current-work/html5-letter.pdf {
-            return 301 https://html.spec.whatwg.org/print.pdf;
-        }
-        location = /specs/web-apps/current-work/webrtc.html {
-            return 301 https://w3c.github.io/webrtc-pc/;
-        }
-        location = /specs/web-apps/current-work/websrt.html {
-            return 301 https://w3c.github.io/webvtt/;
-        }
-        location = /specs/web-apps/current-work/webvtt.html {
-            return 301 https://w3c.github.io/webvtt/;
-        }
-        # fall back to single-page HTML
+    location = /specs/web-apps/current-work/complete.html {
```

Comment:
This needs to be added back because it was previously handled by "fall back to single-page HTML", but not any longer.